### PR TITLE
[TEST] Skip testing triangular_solve_batched on non-default CUDA stream

### PR DESCRIPTION
**Pull request by vishwakftw**
This is for testing purposes.

### CODE REVIEW

**Comment by vishwakftw** [comment]
@zou3519, it seems like non-default CUDA stream testing caused this to fail.

**Comment by zou3519** [comment]
Sure, let's get this merged and file an issue for triangular_solve not working on non-default stream. Thank you for investigating!

**Review by zou3519 — APPROVED** [review]

File: test/test_cuda.py [hunk 1]
```diff
@@ -2784,7 +2784,7 @@ def test_triangular_solve(self):
         _TestTorchMixin._test_triangular_solve(self, lambda t: t.cuda())
 
     @unittest.skipIf(not TEST_MAGMA, "no MAGMA library detected")
-    @unittest.skip("Spuriously failing")
+    @skipCUDANonDefaultStreamIf(True)
```
Comment (zou3519) [inline test/test_cuda.py]: If we're going to skip this we should put a message as to why (i.e., tringular_solve doesn't work with non default stream, or there is something wrong with the non-default stream testing) and link to an issue.

Comment (vishwakftw) [reply test/test_cuda.py]: I believe triangular_solve doesn't work with non-default stream. This is an issue that takes this generally.

Comment (vishwakftw) [reply test/test_cuda.py]: `test_triangular_solve` is also skipped in an analogous manner.

**Comment by zou3519** [comment]
@vishwakftw just checking, do you want this merged (so that we can prevent regressions) or are you still using this as a testing ground?

**Comment by vishwakftw** [comment]
This can be merged, but I think that @mruberry’s PR on test refactor may have superseded this.

**Comment by mruberry** [comment]
You can merge as is.

**Comment by zou3519** [comment]
Cool, I'll merge this as is. Thanks for the discussion!

**Pull request closed without merging**